atomic_reactor/plugins/post_rpmqa.py

```diff
@@ -70,3 +87,45 @@ def gather_output(self):
                 return output
 
         raise RuntimeError('Unable to gather list of installed packages in container')
+
+    def gather_output_scratch(self):
+        container_dict = self.tasker.create_container(self.image_id, command=['/bin/bash'])
+        container_id = container_dict['Id']
+        self._container_ids.append(container_id)
+
+        try:
+            bits, _ = self.tasker.get_archive(container_id, RPMDB_PATH)
+        except APIError as ex:
+            self.log.info('Could not extract rpmdb in %s : %s', RPMDB_PATH, ex)
+            return None
```

Comment:
when does this happen, should we not fail the build as well?

Reply:
when path isn't in the image => rpmdb doesn't exist

in that case we just want to finish without failing

Reply:
I see now, since list_rpms_from_scratch is true by default we don't want to affect from scratch images without rpms.